web/app/controllers/dashboard_controller.rb

```diff
@@ -22,7 +25,34 @@ def index
     end
   end
 
+  def get_application_name(app_id)
+    req = Soapbox::GetApplicationRequest.new(id: app_id)
+    @app = $api_client.get_application(req)
+    @app.name
+  end
+
+  helper_method :get_application_name
+  helper_method :get_environment_name
+  helper_method :get_user_name
```

Comment:
These `helper_method` calls should go to the top underneath the `before_action` call